solutions/3522/01.py

```
@@ -1,15 +1,17 @@
 class Solution:
-    def resultsArray(self, nums: List[int], k: int) -> List[int]:
-        n = len(nums)
-        res = []
+    def calculateScore(self, instructions: List[str], values: List[int]) -> int:
```

Ruff (0.14.10)

Undefined name `List`

(F821)